app/views/merchant_invoices/show.html.erb

```diff
@@ -33,6 +33,12 @@
         <p>Status: <%= @invoice.get_invoice_item(item.id).status %></p>
       </li>
     </ul>
+    <%= form_with url: "/merchants/#{@invoice.get_invoice_item(item.id).item.merchant_id}/invoices/#{@invoice.id}?item_id=#{item.id}", method: :patch, local: true do |form| %>
+      <%= form.label :select_status %>
+      <%= form.select :select_status,
+                      ['pending', 'packaged', 'shipped'] %>
+      <%= form.submit 'Update Item Status' %>
+    <% end %>
```

Comment:
This select form look nicceeeeee. I like the setup. 
Does the user story want to have the current status selected by default? If so, that would be my only recommendation.

Reply:
Good catch! I'll add that now and push it up

Reply:
Refactor looking good. Ready for merge.